src/util/prop-definition.js

```diff
@@ -0,0 +1,104 @@
+import dashCase from './dash-case';
+
+/**
+ * @internal
+ * Property Definition
+ *
+ * Internal meta data and strategies for a property.
+ * Created from the options of a PropOptions config object.
+ *
+ * Once created a PropDefinition should be treated as immutable and final.
+ * 'getPropsMap' function memoizes PropDefinitions by Component's Class.
+ *
+ * The 'attribute' option is normalized into the 'attrName' property.
+ */
+export default class PropDefinition {
+
+  // constructor(name:string|symbol, cfg:PropOptions) {
+  constructor (nameOrSymbol, cfg) {
+    this._name = nameOrSymbol;
+
+    cfg = cfg || {};
+
+    if (typeof cfg === 'function') {
+      // todo: Where is documented that a config can just be the coerce function?
```

Comment:
I don't think it is. This is an artefact of the 0.x API, I think. Happy for you to remove this in a minor as it was never documented in the major version APIs.